pkg/virt-handler/non-root.go

```diff
@@ -108,101 +103,6 @@ func (d *VirtualMachineController) prepareStorage(vmi *v1.VirtualMachineInstance
 	return changeOwnershipOfHostDisks(vmi, res)
 }
 
-func getTapDevices(clusterConfig *virtconfig.ClusterConfig, vmi *v1.VirtualMachineInstance, networkBindings map[string]v1.InterfaceBindingPlugin) (map[string]string, error) {
-	tapNetworks := map[string]struct{}{}
-	domainAttachmentByInterfaceName := domainspec.DomainAttachmentByInterfaceName(vmi.Spec.Domain.Devices.Interfaces, networkBindings)
-	for _, inf := range vmi.Spec.Domain.Devices.Interfaces {
-		if domainAttachmentByInterfaceName[inf.Name] == string(v1.Tap) {
-			tapNetworks[inf.Name] = struct{}{}
-		}
-	}
```

Comment:
This seems to have been processed for the `macvtap` binding.
Or in more general terms, for domain-attachment-type of `tap`. I.e. a plugin that created the Tap already, like the macvtap one.

However, I think we have e2e tests for the macvtap plugin, and they seem to pass.